docs-ref-conceptual/azure-cli-support-lifecycle.md

```diff
@@ -0,0 +1,194 @@
+
+Azure CLI **Core** is updated monthly with a maximum of two releases introducing breaking changes. Breaking change releases occur during the second and fourth quarters of each year. 
+
+Azure CLI **extensions** don't have a specific release cadence and need to be upgraded manually. 
+
+Breaking changes might not affect existing scripts, but it's a good practice to verify your scripts in a test environment before performing an upgrade. 
+
+> [!NOTE]
+> The release cadence might be slightly adjusted to be aligned with major announcements relevant to Azure.
+
+## Standard Term Support releases (STS)
+
+The STS versions of Azure CLI core give you the most recent capabilities. STS releases don't experience breaking changes for six months. Breaking changes are released in the second and fourth quarters, and usually align with Microsoft Build and Microsoft Ignite.
+
+STS releases have the following characteristics:
+
+* STS releases are released monthly and receive feature updates, bug fixes, and security fixes.  
+* To be supported, you need to be on the latest minor version of the STS release.
+* STS releases don't have a specific marker when running the `az version` command. This ensures backward compatibility with tools or scripts that might utilize the information.
+
+## Long Term Support releases (LTS)
+
+The LTS versions of Azure CLI Core give you a "maintenance" period of 12 months following the "current" period. During this time, the LTS release doesn't experience breaking changes _unless required by a security fix_.  
+
+The LTS versions are based on Azure CLI releases that introduce major changes during the second quarter of the calendar year. This usually aligns with Microsoft Build.  
+
+LTS releases have the following characteristics:
+
+* During the first six months, the LTS releases are released monthly and receive feature updates, bug fixes and security fixes.
+* During the following 12 months, the LTS releases are released when necessary and only receive bug fixes and security fixes.  
+* To be supported, you need to be on the latest version of the LTS release.  
+
+Easily find the LTS release date by running the `az version` command. Notice the `azure-cli-support` line that indicates the LTS date.
+
+```output
+{  
+  "azure-cli": "2.48.0",  
+  "azure-cli-core": "2.48.0",  
+  "azure-cli-support": "LTS-20240521",  
+  "azure-cli-telemetry": "1.0.8",  
+  "extensions": {  
+  }  
+}  
+```
+
+The following image illustrates the release cadence and support lifecycle of Azure PowerShell for LTS and STS releases.
+
+![Support Lifecycle](./media/support-lifecycle/support-lifecycle.png)
+
+The following table provides examples of how the release cadence correlates with the support
+lifecycle:
+
+> [!NOTE]
+> The dates in the following table are for example purposes only. The actual release dates vary.
+
+| Version number | Release schedule | Breaking changes | Release type | Support type | Supported until version |
```

Comment:
@dcaro , I replaced my table with Mikes; however, this dropped the patch versions I had illustrated. I know we wanted azCLI and azPS to use the same release cadense, but is this chart correct for azCLI?